FAQ_CN.md

```diff
@@ -220,3 +218,25 @@ curl http://127.0.0.1:9080/apisix/admin/plugins/reload -H 'X-API-KEY: edd1c9f034
 ```shell
 apisix reload
 ```
+
+## 如何格式化中文文档？
+
+中文文档对于格式有很多要求，比如英文单词前后要有空格` `，但每次手动修改都非常繁琐，有没有省时省力的办法解决这个问题？
+
+推荐使用 vscode 的 pangu 插件
+
+**安装方式**
+
+进入 vscode 插件列表，搜索`pangu`，安装`Pangu-Markdown`插件
+
+![pangu](doc/images/pangu-plugin-1.png)
+
+**使用方式**
+
+进入 vscode 命令行模式，选择 `Pangu Format`
```

Comment:
use `Command + P` to show the commands panel on Mac?

Reply:
shift + command + p